formbuilder/config.js

```diff
@@ -229,8 +229,18 @@ export default {
             title: {type: "string", title: "Label"},
             description: {type: "string", title: "Description"},
             required: {type: "boolean"},
-          }
+          },
```

Comment:
Maybe we want to comment from line https://github.com/Kinto/formbuilder/pull/57/files#diff-53ad1f93140dfed2662a5448dfa859a9R207